src/Service/BreadcrumbItemProcessor.php

```diff
@@ -78,7 +79,12 @@ private function processItem(BreadcrumbItem $item, array $variables): ProcessedB
     {
         // Process the label
         if ($item->label && \str_starts_with($item->label, '$')) {
-            $processedLabel = $this->parseValue($item->label, $variables);
+            $labelValue = $this->parseValue($item->label, $variables);
+            if ($labelValue instanceof TranslatableMessage) {
+                $processedLabel = $labelValue->trans($this->translator, $item->translationDomain ?: null);
```

Comment:
Based on `TranslatableMessage::trans()`, the second parameter is the translation locale (optional). So, passing the translation domain here is incorrect (also because an eventual domain would be provided by TranslatableMessage itself).

Also, could you also add a test and update the documentation to describe this new feature? Thanks!